Sources/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.m

```diff
@@ -0,0 +1,23 @@
+#import "QuickSpec+QuickSpec_MethodList.h"
+#import <objc/runtime.h>
+
+
+@implementation QuickSpec (QuickSpec_MethodList)
+
++ (NSSet<NSString*> *)allSelectors {
+    id t = [[[self class] alloc] init];
+    NSMutableSet<NSString*> *allSelectors = [NSMutableSet set];
+    
+    unsigned int methodCount = 0;
+    Method * mlist = class_copyMethodList(object_getClass(t), &methodCount);
+    NSLog(@"%d methods", methodCount);
+    for(int i = 0; i<methodCount; i++) {
```

Comment:
Nit pick: How about an `unsigned int` for the index here? It matches the type of `methodCount` that way. Also, add a space in between `i`, `<`, and `methodCount` -- as in `i < methodCount`.